packages/core/src/spam-protection/google-recaptcha.ts

```diff
@@ -66,16 +66,28 @@ export default class GoogleRecaptcha {
     }
 
     load(containerId: string, sitekey: string): Promise<void> {
-        return this.googleRecaptchaScriptLoader.load().then((recaptcha) => {
-            if (recaptcha) {
-                this._event$ = this._memoized(
-                    recaptcha,
-                    sitekey,
-                    document.getElementById(containerId),
-                );
-                this._recaptcha = recaptcha;
-            }
-        });
+        return this.googleRecaptchaScriptLoader
+            .load()
+            .then((recaptcha) => {
+                if (recaptcha) {
+                    this._event$ = this._memoized(
+                        recaptcha,
+                        sitekey,
+                        document.getElementById(containerId),
+                    );
+                    this._recaptcha = recaptcha;
+                }
+            })
+            .catch((err) => {
+                throw err;
+            });
```

Comment:
I see we're adding this block here.
Could you please explain how the error will be handled?

Reply:
This should be on the clients or just throw. It was previously unhandled so made this change.